### PR TITLE
Create 140.Word Break II

### PR DESCRIPTION
**Pull request by kwang18**
@tianheng

### CODE REVIEW

File: 140.Word Break II [hunk 1]
```diff
@@ -0,0 +1,25 @@
+using CanBreakAndStringSoFar = pair<bool, vector<string>>;
```
Comment (tianheng) [inline 140.Word Break II]: put it inside of class

**Review by tianheng — COMMENTED** [review]
Need a detailed analysis for time and space.

**Comment by kwang18** [comment]
@tianheng

**Review by tianheng — CHANGES_REQUESTED** [review]
Comments as above,

**Pull request merged**